amida-tech/amida-notification-microservice
# Adding autocreate to db and .env.test

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
Adds .env.test for use in testing environment 
Creates database on testing and starting of the app.
Fixes a bunch of linting errors
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/SER-229
https://jira.amida-tech.com/browse/SER-222
#### How should this be manually tested?
run `yarn build`
create `.env.test` file
change the db name in `.env.test` file
run `yarn create_db`
run `yarn test` verify it FAILS 
run `yarn delete_db`
run `yarn test` verify it works

#### Any background context you want to provide?
#### Screenshots (if appropriate):
